pkgs/development/python-modules/pydmd/default.nix

```
@@ -6,53 +6,59 @@
 , matplotlib
 , numpy
 , pytestCheckHook
+, pytest-mock
 , pythonOlder
 , scipy
 , ezyrb
 }:
 
-buildPythonPackage rec {
-  pname = "pydmd";
-  version = "0.4.0.post2302";
-  format = "setuptools";
+let
+  self = buildPythonPackage rec {
+    pname = "pydmd";
+    version = "1.0.0";
+    setuptools = true;
```

```suggestion
    pyproject = true;
```
Please use `pypeoject` and ...